src/calculateRentalCost.js

```diff
@@ -1,10 +1,26 @@
 /**
- * @param {number} days
+ * Calculates the rental cost based on the number of days rented,
+ * applying different discounts based on the duration.
  *
- * @return {number}
+ * @param {number} days - The number of days the item is rented for.
+ * @return {number} The total rental cost.
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAILY_RATE = 40;
+  const BASIC_DISCOUNT = 20;
+  const ADDITIONAL_DISCOUNT = 50;
+  const BASIC_DISCOUNT_THRESHOLD = 3;
+  const ADDITIONAL_DISCOUNT_THRESHOLD = 7;
+
+  if (days < BASIC_DISCOUNT_THRESHOLD) {
```

Comment:
The condition 'days < BASIC_DISCOUNT_THRESHOLD' will not apply the discount to rentals of exactly 3 days, which is not in accordance with the problem statement. Consider changing '<' to '<='.